### ISSUE TITLE
resize Linode does not accept allow_auto_disk_resize

### ISSUE THREAD

**Issue by rahulbhatu** [linode/linode_api4-python#180]
Hello All,

I am trying to resize a linode so i tried with cli and it worked well, cli allows to pass optional parameter allow_auto_disk_resize when doing resize, 

Example :
`linode-cli linodes resize 123456 --type g6-standard-1 --allow_auto_disk_resize false
`
I am trying to perform same operation with this linode_api4 but seems like it does not accept the allow_auto_disk_resize.
can someone help me confirm this if this is a limitation or i am missing something. 

below is the code snippet i am using. which works but when i add allow_auto_disk_resize=False it fails
```
from linode_api4 import LinodeClient

token = '123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890ABCDEFGHIJKLMNOPQRS'
client = LinodeClient(token)
loaded_linode = client.load(Instance, 12345678)

resize = loaded_linode.resize(new_type="g6-standard-4")
```
This does not work. and i end up resizing my disk which i dont want.
```
from linode_api4 import LinodeClient

token = '123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890ABCDEFGHIJKLMNOPQRS'
client = LinodeClient(token)
loaded_linode = client.load(Instance, 12345678)

resize = loaded_linode.resize(new_type="g6-standard-4", allow_auto_disk_resize=False)

```

**Comment by Dorthu** [comment]
Thanks for the report - I'm looking at this now.

**Comment by Dorthu** [comment]
This was fixed in release 3.0.1, which is on pypi right now.  A `pip install --upgrade linode_api4` should get it fixed for you.  Thanks again for the report!

**Comment by rahulbhatu** [comment]
@Dorthu 
Thank you so much for such a quick response and fix. I will test it right away and check.
